mc_interface.c

```diff
@@ -1922,7 +1922,7 @@ static void update_override_limits(volatile motor_if_state_t *motor, volatile mc
 	float lo_max_mot = l_current_max_tmp;
 	if (motor->m_temp_motor < conf->l_temp_motor_start) {
 		// Keep values
-	} else if (motor->m_temp_motor > (conf->l_temp_motor_end + 10.0)) { // only fault if 10 degrees over max do give some cushion for noisey sensors
+	} else if (motor->m_temp_motor > conf->l_temp_motor_end) { // only fault if 10 degrees over max do give some cushion for noisey sensors
```

Comment:
Sorry for being kind of fussy, but could you also remove the comment for less confusion ?

Reply:
not fussy at all thanks for the catch